actpool/actpool.go

```diff
@@ -23,6 +24,35 @@ import (
 	"github.com/iotexproject/iotex-core/pkg/log"
 )
 
+var (
+	actpoolMtc = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "iotex_actpool_metrics",
+			Help: "actpool metrics.",
+		},
+		[]string{"type", "value"},
+	)
+	failedToGetAddress     = "failed to get address from bytes"
```

Comment:
Can we shorten the description as it's not served as the log? This will be the legends for graphana charts